src/js/components/FormField/FormField.js

```diff
@@ -474,14 +476,19 @@ const FormField = forwardRef(
                 event.persist();
                 if (onChange) onChange(event);
                 if (contextOnChange) {
-                  const debouncedFn = debounce(() => {
-                    contextOnChange(event);
-                    // A half second (500ms) debounce can be a helpful starting
-                    // point. You want to give the user time to fill out a
-                    // field, but capture their attention before they move on
-                    // past it. 2 second (2000ms) might be too long depending
-                    // on how fast people type, and 200ms would be an eye blink
-                  }, 500);
+                  const debouncedFn = debounce(
+                    () => {
+                      contextOnChange(event);
+                      // A half second (500ms) debounce can be a helpful
+                      // starting point. You want to give the user time to
+                      // fill out a field, but capture their attention before
+                      // they move on past it. 2 second (2000ms) might be too
+                      // long depending on how fast people type, and 200ms
+                      // would be an eye blink
+                    },
+                    500,
```

Comment:
Wondering if this delay should be something the caller can configure? Especially since we don't know what kind of validation function might be called. There is a [theme prop](https://github.com/grommet/grommet/blob/f8c7358c43895ac8bb10e91ada33a5356a3adedd/src/js/themes/base.js#L223), but seems to have been abandoned...

Reply:
Yes, I agree that this should be configurable. Since `debounceDelay` is not being used currently I think we could change it from '300' to '500' and then use it here